sphinxcontrib/mermaid.py

```diff
@@ -415,15 +415,24 @@ def install_js(
     if doctree and not doctree.next_node(mermaid):
         return
 
+    # Add required JavaScript
     # Add required JavaScript
     if not app.config.mermaid_version:
         _mermaid_js_url = None  # assume it is local
     elif app.config.mermaid_version == "latest":
-        _mermaid_js_url = "https://unpkg.com/mermaid/dist/mermaid.min.js"
+        _mermaid_js_url = "https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.esm.min.mjs"
     else:
-        _mermaid_js_url = f"https://unpkg.com/mermaid@{app.config.mermaid_version}/dist/mermaid.min.js"
+        _mermaid_js_url = f"https://cdn.jsdelivr.net/npm/mermaid@{app.config.mermaid_version}/dist/mermaid.esm.min.mjs"
     if _mermaid_js_url:
-        app.add_js_file(_mermaid_js_url, priority=app.config.mermaid_js_priority)
+        # Configuration for Mermaid initialization
+        mermaid_init_script = (
+            f"import mermaid from '{_mermaid_js_url}';\n"
+            "let config = { startOnLoad: true };\n"
+            "mermaid.initialize(config);"
```

Comment:
we should support customization via `app.config.mermaid_init_js` here. change the default if needed (update the README)